types/wlr_data_device.c

```diff
@@ -433,7 +415,7 @@ static void wlr_drag_set_focus(struct wlr_drag *drag,
 		if (wl_resource_get_version(offer->resource) >=
 				WL_DATA_OFFER_SOURCE_ACTIONS_SINCE_VERSION) {
```

Comment:
Seems like you got rid of the version check everywhere but here. Will it crash a client if we send actions and they don't support it?

Reply:
Oh I see where the checks are now. This seems to be for the offer.